src/handlers/course.handler.ts

```
@@ -15,14 +15,37 @@ export class CourseHandler {
+      console.error('Error handling user course add:', error);
+      throw error;
+    }
+  }
+
+  async handleCourseEnrollmentCreated(data: any) {
+    try {
+      console.log('Course enrollment created:', data);
+      // Fetch course details to get course name
+      const courseDetails = await this.getCourseName(data.courseId);
+      const courseName = courseDetails.result.content.name || 'Unknown Course';
+      
```

ESLint

[error] Delete `······`

(prettier/prettier)